insight-rest-objects/src/test/resources/tensor_instances.json

```diff
@@ -0,0 +1,18 @@
+{
+  "docId": "aDocumentId",
+  "instances": [
+    {
+      "dc:subjects": {
+        "type": "cat",
+        "categories": [
+          "sciences",
+          "art/cinema"
+        ]
+      },
+      "dc:title": {
+        "type": "txt",
+        "text": "My document title"
+      }
+    }
+  ]
+}
```

Comment:
In the future add an image as example. For now maybe no need.

Reply:
those are the tests on serialization.

Reply:
yes, to test image serialization also.

Reply:
this client doesn't serialize an image. It doesn't know of Nuxeo internals. It's JSON -> Object not JSON -> Nuxeo Entity